### ISSUE TITLE
Enum in migration not working (Postgres)

### ISSUE THREAD

**Issue by Marlysson** [MasoniteFramework/orm#359]
**Describe the bug**
A migration using enum type in postgres database isn't working.

**To Reproduce**
1. Create a migration with `table.enum('gender', ['male', 'female'])`
2. Run migration

**Expected behavior**
Constraint check is have to create normally.

**Screenshots**
![image](https://user-images.githubusercontent.com/4117999/106619991-e85a8400-654f-11eb-9754-3fb668cce797.png)

**What database are you using?**
Postgres

**Comment by josephmancuso** [comment]
I think the proper syntax is:

```
   CHECK (element_type IN ('mail', 'female') );
```

**Comment by josephmancuso** [comment]
Actually no. This is the proper way to create enums in Postgres we can do:

```
CREATE TYPE users10_genders_enum AS ENUM ('male', 'female');

create table users10(
    genders users10_genders_enum
);
```

**Comment by josephmancuso** [comment]
Another way all in one line and probably easier is to do this:

```python
create table users11(
    genders VARCHAR(255) CHECK (genders = 'male' OR genders = 'female')
);
```

This just involves changing the `male, female` syntax to `genders = 'male' OR genders = 'female'`

**Comment by josephmancuso** [comment]
Lastly we can also do this. This requires to smallest change:

```sql
create table users12(
    genders VARCHAR(255) CHECK (genders IN ('male', 'female'))
);
```

This just requires adding `{column} IN ` to the query

**Comment by josephmancuso** [comment]
postgres has so many damn ways to do this

**Comment by Marlysson** [comment]
This seems the better way:
> genders VARCHAR(255) CHECK (genders IN ('male', 'female'))

I have a postgres installed, I can test against a database this behavior. Or do you think define this as a `good first issue` issue?

**Comment by josephmancuso** [comment]
no this needs to be fixed ASAP

**Comment by josephmancuso** [comment]
Fixed in #363
